source/includes/APIReference/CustomFields/_create.md.erb

```diff
@@ -0,0 +1,40 @@
+## Create Custom Fields
+
+<%= partial "includes/APIReference/CustomFields/Examples/create.tmpl.erb" %>
+
+Add one or more customfields.
+
+### HTTP Request
+
+<img src="images/post.png" alt="post"/><api>https://rest.tsheets.com/api/v1/customfields</api>
+
+### Properties
+_Pass an array of customfield objects as the value to a 'data' property (see example)._
+
+|                |             |             |
+| -------------: | :---------: | ----------- |
+| **name**<br/>required | _String_ | The name of this customfield. |
+| **required**<br/>optional | _Boolean_ | If true, a non-null value must be selected for this customfield when it appears on a timesheet. |
+| **active**<br/>optional | _Boolean_ | If false, this `customfield` is considered archived and will not be visible or usable. |
+| **applies_to**<br/>required | _String_ | Allowed values: 'timesheet'. Specify that this `customfield` should appear on timesheets (additional customfield types may be supported in the future). |
+| **type**<br/>required | _String_ | Allowed values: 'managed-list', 'free-form'.  A managed-list customfield is intended to be displayed as a dropdown list, where each 'option' is a customfielditem. A 'free-form' customfield supports text entry and does not require a collection of customfielditems to accompany it. |
+| **show_to_all**<br/>optional | _Boolean_ | Default: `true`.  Declares whether this customfield should exist on timesheets for all workers. If false, it will only appear on timesheets for workers to whom it has been assigned. |
+| **short_code**<br/>optional | _String_ | This is a shortened code or alias that is associated with the customfield. It may only consist of letters and numbers.
+
+For a full list of the properties that may be set on a customfield, see [The Custom Field Object](#the-custom-field-object).
```

Comment:
I'll test out some different verbiage here, maybe:

> For a full list of properties that exist on a customfield, see [The Custom Field Object](#the-custom-field-object).

Reply:
Yeah, I like your suggested change, @evanhsu

Reply:
Would we also want to add `show_to_all` to the Custom Field Object, since from my understanding, and how I coded it (subject to change), it should be editable on the PUT as well?

Reply:
I've added the `show_to_all` property to the Custom Field Object as a `read-write` property